apps/estuary/lib/estuary/helper/kafka_helper.ex

```diff
@@ -0,0 +1,5 @@
+defmodule Estuary.KafkaHelper do
+  def list_exisiting_topics do
+    Elsa.list(localhost: 9092)
```

Comment:
May be worth doing an IO.inspect on this guy to see the output when you call this (I'm assuming little test) function.

Reply:
yes